### PR TITLE
Add support for EDAM data annotations to datatypes

### PR DESCRIPTION
**Pull request by nsoranzo**
This is an outcome of the [Technical Hackathon: Tools, Workflows and Workbenches](https://www.eventbrite.fr/e/technical-hackathon-tools-workflows-and-workbenches-tickets-22797525011) and contains work by @bgruening, @hmenager, @mvdbeek and me.

It contains a subset of PR #1891 which should be immediately mergeable.

### CODE REVIEW

File: lib/galaxy/datatypes/assembly.py [hunk 1]
```diff
@@ -19,7 +19,9 @@
 
 class Amos( data.Text ):
     """Class describing the AMOS assembly file """
-    edam_format = "format_2561"
+
```
Comment (mvdbeek) [inline lib/galaxy/datatypes/assembly.py]: let's remove that blank line

**Comment by bgruening** [comment]
Lovely! Thanks so much for finishing it!
I would remove some blank-lines but this is just my taste. Let's merge this if the tests are passing!

Thanks again!

**Comment by nsoranzo** [comment]
@bgruening Blank lines removed.

**Comment by bgruening** [comment]
Great work!

**Comment by hmenager** [comment]
Thank you so much everyone for getting this done! Great team work indeed!

**Comment by jmchilton** [comment]
Awesome sauce - thanks all!

**Pull request merged**